ciscoise_consts.py

```diff
@@ -33,9 +33,11 @@
 ERS_POLICIES = ":9060/ers/config/ancpolicy"
 
 # Error/Success
-CISCOISE_ERROR_TEST_CONNECTIVITY_FAILED = "Test connectivity failed"
+CISCOISE_ERROR_TEST_CONNECTIVITY_FAILED_1 = "Test connectivity failed for primary device"
```

Comment:
"_1" and "_2" are a bit enigmatic, when reading the code of the connector.  Maybe we can consider replacing these suffixes with "_PRIMARY" and "_SECONDARY".